tests/qm-connectivity/main.fmf

```diff
@@ -0,0 +1,12 @@
+summary: Concise summary describing what the test does
```

Comment:
Also suggest adding comment w/ link to format docs.

Reply:
Ack